carbon-design-system/carbon
# refactor(Button): padding units from pixel to rem

Closes #10318 

Change remaining `px` units in `Button` padding to `rem`

#### Changelog

**New**

- n/a

**Changed**

- update padding pixel units to rem equivalent

**Removed**

- n/a

#### Testing / Reviewing

1. Confirm pixel values are now in `rem`
2. Confirm the new `rem` calculations still have the same desired size outcome as before. Below are some Dev Tool images of the original button sizes before this update for comparison.
- sm
![Padding](https://user-images.githubusercontent.com/9935383/228008681-c08a6170-657b-437c-af9e-644d91fa9095.png)

- md
![image](https://user-images.githubusercontent.com/9935383/228008709-8e8e7c3f-92d5-41b0-9a6d-440102f9b024.png)

- lg
![Padding](https://user-images.githubusercontent.com/9935383/228008730-d08d5f89-22e5-4cfb-b598-4628dd9dfb84.png)

- xl
![Padding](https://user-images.githubusercontent.com/9935383/228008766-7b171cb0-73d7-4a60-9cd4-f243ee1dd247.png)

- 2xl
![Padding](https://user-images.githubusercontent.com/9935383/228008799-a0882b4e-503c-4134-ade3-1fb8cc8aa921.png)

3. Here are some values you can paste into `Secondary` buttons in `Button.stories.js` to manually inspect the values per `size`.
```
<>
    <Button size="sm" kind="secondary" {...args}> Button </Button>
    <Button size="md" kind="secondary" {...args}> Button </Button>
    <Button size="lg" kind="secondary" {...args}> Button </Button>
    <Button size="xl" kind="secondary" {...args}> Button </Button>
    <Button size="2xl" kind="secondary" {...args}> Button </Button>
</>
```
